### ISSUE TITLE
upgrade redis version

### ISSUE THREAD

**Issue by kayiwa** [DocNow/docnow-ansible#61]
The latest version of redis will allow us to render videos

**Comment by edsu** [comment]
Thanks! The video fetching code relies on `LPOP`'s `count` argument, which is available in Redis >= v6.2.0. We currently have v4.0.9 installed on smoketest. Would it be possible to get the latest stable Redis in our DocNow playbook?

**Comment by edsu** [comment]
It looks like we will need the [Redis PPA](https://redis.io/download#from-the-official-ubuntu-ppa)?

```
$ sudo add-apt-repository ppa:redislabs/redis
$ sudo apt-get update
$ sudo apt-get install redis
```
